packages/framework-provider-aws/src/library/searcher-adapter.ts

```diff
@@ -9,7 +10,7 @@ export async function searchReadModel(
   config: BoosterConfig,
   logger: Logger,
   readModelName: string,
-  filters: Record<string, Filter<any>>
+  filters: FilterFor<any>
```

Comment:
Can we use `unknown` instead of `any` here?

Reply:
What would be the difference here @NickSeagull? I'm still learning some things about TS :)

Reply:
The `unknown` type forces the programmer to cast the type to the appropriate type when its needed, while `any` can be passed around freely:

![](https://pbs.twimg.com/media/Ehw9hCvWAAUF8ct?format=jpg&name=900x900)